README.md

```diff
@@ -0,0 +1 @@
+This repo is about to test code reviews.
```

Comment:
Kindly ensure that the README file looks more professional and more descriptive about your project.

Reply:
- [Optional] Kindly use a couple of emojis and/or gifs in the review.
They make the code review more human-friendly 👽🤖.